Slack-Apps/week3/homework/responses/3.3-calling-translator-api.md

```diff
@@ -0,0 +1,90 @@
+# Google Translate API: Calling the Translator
+
+You should now return to your ```app.js``` to implement adding Cloud Translation. In the livestream, you should have noticed the Slack App response was hardcoded to a phrase like "This is Spanish" or "This is Japanese". Now, we are going to be adding the response as the actual ```translated text``` in a thread.
+
+## Integrating the API
+
+In the first issue (3.1), you should have already used ```.includes``` and ```.split(-)``` to find the country code of the flag and if not, created a langMap. It should have looked something like this: ```const [x, country] = langMap[reaction] ? [, reaction] : reaction.split('-');```
+
+We also added the feature of responding to a thread but which thread you may ask? Well, we need to find that. To do so, we are going to set ```const result``` equal to ```await client.conversations.history({ Edit Here })```.
```

Comment:
maybe explain the syntax of ```await client.conversations.history({ Edit Here })``` a bit more if you haven't yet previously instead of just saying that this is how to get the value. What does "conversations" refer to/contain? How about history?